src/planning/include/planning/track.hpp

```diff
@@ -41,6 +43,30 @@ class Track {
    */
   ~Track();
 
+  /**
+   * @brief log cones on one side of the track
+   *
+   * @param side if true -> left, false -> right
+  */
+  void logCones(bool side);
+
+  /**
+   * @brief rounds numbers to n decimal places
+   * 
+   * @param num number to be rounded
+   * @param decimal_places number of decimal places
+   * @return rounded number
+   */
+  float round_n(float num, int decimal_places);
```

Comment:
This method is not related to this class whatsoever. As such, it should be in another place, maybe an utils file

Reply:
> This method is not related to this class whatsoever. As such, it should be in another place, maybe an utils file

Since it is used for testing only I'll place it in the tests.cpp file